src/sentry/tasks/integrations/migrate_issues.py

```diff
@@ -0,0 +1,53 @@
+from django.db import IntegrityError, transaction
+from rest_framework.response import Response
+
+from sentry.models import ExternalIssue, Group, GroupLink, GroupMeta, Integration
+from sentry.tasks.base import instrumented_task, retry
+from sentry.tasks.integrations import logger
+
+
+@instrumented_task(
+    name="sentry.tasks.integrations.migrate_issues",
+    queue="integrations",
+    default_retry_delay=60 * 5,
+    max_retries=5,
+)
+@retry(exclude=(Integration.DoesNotExist))
+def migrate_issues(
+    integration_id: int, organization_id: int, project_id: int, plugin_slug: str
+) -> None:
+    groups = Group.objects.filter(project=project_id)
+    # this seems expensive but I don't think there are other options
+    plugin_issues = GroupMeta.objects.filter(
+        key=f"{plugin_slug}:tid", group__id__in=[group.id for group in groups]
+    )
```

Comment:
This pattern for look up is unique to Jira right? Do we want to pass this in as a param instead, or make this task specific to Jira in its name?

Reply:
No, it's not unique. For example there is also `bitbucket:tid`, `github:tid`, `gitlab:tid` etc. idk what `tid` stands for though 🤔